README.md

```diff
@@ -89,6 +90,30 @@ When grouping protocol conformance, always use the name of the protocol and only
 // MARK: Table View Delegate
 ```
 
+###Guard Statements
+
+#####Single assignment `guard`
+```Swift
+guard let value = someMethodThatReturnsOptional() else {
```

Comment:
I would love to see some note here about using guard as early return only, and not nested in and out of regular control logic.